namer/core/src/main/scala/io/buoyant/namer/DelegatingNameInterpreter.scala

```diff
@@ -0,0 +1,21 @@
+package io.buoyant.namer
+
+import com.twitter.finagle.naming.NameInterpreter
+import com.twitter.finagle._
+import com.twitter.util.Activity
+
+trait DelegatingNameInterpreter extends NameInterpreter {
```

Comment:
Does this need to extend NameInterpreter? I think this trait could simply express delegation features, i.e. `Delegator`.  The NameInterpreter-ness of this type isn't really related to Delegator-ness.  `ConfiguredNamersInterpreter` would extend `NameInterpreter with Delegator`, and `DelegateApiHandler` can take a `String => Option[Delegator]`... tioli.